packages/node-cli/README.md

```diff
@@ -25,9 +25,7 @@ Each version of CLI for Node.js in NPM works optimally with a specific NetSuite
 
 | Version in NPM | Version in NetSuite |
 |:--------------:|:-------------------:|
-| 1.2.X | 2021.1 |
```

Comment:
Why are we deleting the older versions.
Maybe there is a miscommunication. I explained David when i met him that we shouldn't delete the existing npm versions from the table.
Was this request from David?

Reply:
Mmmm I checked previous Release Notes for Node in NetSuite Help Center and they do not include previous versions. That is why I deleted them.

Reply:
Kindly see attached screenshot 
<img width="258" alt="Capture" src="https://user-images.githubusercontent.com/87807151/128872979-7b99f54a-ef82-4254-afb1-9e7bfb047590.PNG">
https://tstdrv2472202.app.netsuite.com/app/help/helpcenter.nl?fid=section_1558730192.html#subsect_161035969554

Reply:
So these are 2 different documents.
Release notes are specific for whats new in the release. And they are documented in help of Netsuite.

This readme file is not release specific.
Its an overall guide/help for user.
If you check master you will see the format.

Reply:
Oookey then. I will re-add the versions and do the ultimate (I hope) commit.

Reply:
Versions added guys. Kindly review it for me. Thanks for your help!